_templates/generator/help/index.ejs.t

```diff
@@ -0,0 +1,5 @@
+---
+message: |
+  hygen {bold generator new} --name [NAME] --action [ACTION]
+  hygen {bold generator with-prompt} --name [NAME] --action [ACTION]
+---
```

Comment:
@sarahtsige Can you reset these files in the `_templates` directory? Not sure why they're showing up in this pull request.

Reply:
@ZakkMan  by reset, do you mean delete these lines?